storage/client/queries/queries.go

```diff
@@ -286,30 +286,67 @@ const (
 			ORDER BY id DESC`
 
 	ValidatorsData = `
+		WITH self_delegations AS (
+			SELECT delegatee AS address, SUM(shares) AS shares
+				FROM chain.delegations
+				WHERE delegator = delegatee
+				GROUP BY delegatee
+		),
+		delegators_count AS (
+			SELECT delegatee AS address, COUNT(*) AS count
+				FROM chain.delegations
+				GROUP BY delegatee
+		),
+		active_nodes AS (
+			SELECT entities.address, MAX(nodes.voting_power) AS voting_power
+				FROM chain.entities AS entities
+				JOIN chain.nodes AS nodes ON
+					entities.id = nodes.entity_id AND
+					nodes.roles LIKE '%validator%'
+				GROUP BY entities.address
+		)
 		SELECT
-				chain.entities.id AS entity_id,
-				chain.entities.address AS entity_address,
-				chain.nodes.id AS node_address,
-				chain.accounts.escrow_balance_active AS escrow,
-				COALESCE(chain.commissions.schedule, '{}'::JSONB) AS commissions_schedule,
-				EXISTS(SELECT NULL FROM chain.nodes WHERE chain.entities.id = chain.nodes.entity_id AND voting_power > 0) AS active,
-				EXISTS(SELECT NULL FROM chain.nodes WHERE chain.entities.id = chain.nodes.entity_id AND chain.nodes.roles like '%validator%') AS status,
-				chain.entities.meta AS meta
-			FROM chain.entities
-			JOIN chain.accounts ON chain.entities.address = chain.accounts.address
-			LEFT JOIN chain.commissions ON chain.entities.address = chain.commissions.address
-			JOIN chain.nodes ON chain.entities.id = chain.nodes.entity_id
-				AND chain.nodes.roles like '%validator%'
-				AND chain.nodes.voting_power = (
-					SELECT max(voting_power)
-					FROM chain.nodes
-					WHERE chain.entities.id = chain.nodes.entity_id
-						AND chain.nodes.roles like '%validator%'
-				)
-		WHERE ($1::text IS NULL OR chain.entities.address = $1::text)
+			chain.entities.id AS entity_id,
+			chain.entities.address AS entity_address,
+			chain.nodes.id AS node_address,
+			chain.accounts.escrow_balance_active AS active_balance,
+			chain.accounts.escrow_total_shares_active AS active_shares,
+			chain.accounts.escrow_balance_debonding AS debonding_balance,
+			chain.accounts.escrow_total_shares_debonding AS debonding_shares,
+			COALESCE (
+				ROUND(COALESCE(self_delegations.shares, 0) * chain.accounts.escrow_balance_active / NULLIF(chain.accounts.escrow_total_shares_active, 0))
+			, 0) AS self_delegation_balance,
```

Comment:
Could also consider skipping this and have frontend compute this.  Since we return `active_balance`, `active_shares` and `self_delegation_shares` this can be trivially comptued.

But I guess it is also totally fine if we do it here.